independent-publisher-connectors/SOS Inventory/apiProperties.json

```diff
@@ -0,0 +1,33 @@
+{
+	"properties":
```

Comment:
@Harold-Anderson please fix the indentation for apiProperties and apiDefiniton files. Everything should look well-formatted.
Please let me know if I can do this to help you.